app/views/works/top_media.html.erb

```diff
@@ -0,0 +1,67 @@
+
+<section class="spotlight">
+  <div>
+    <h3>
+      <span class="media-spotlight-header">Media Spotlight: <%= @top_medium.title %>
+      </span>
+      by <%= @top_medium.created_by%>
+    </h3>
+  </div>
+
+</section>
+<p>
+</p>
+<h3>
+</h3>
+</div>
+
+<hr class="root-hr">
+
+<body>
```

Comment:
You should not have`body` and `main` inside any of your views since they are already contained without the application.html.erb file and therefore already within all views